tembo-olap/README.md

```diff
@@ -0,0 +1,66 @@
+# Instantiation
+
+Select OLAP Stack from https://cloud.tembo.io
+
+## Choose System Configuration:
+
+**CPU**: 16vCPU
+
+**RAM** : 32GB
+
+**Storage**: 300GB
+
+Instance creation takes *20-25* seconds.
+
+## Connection String
+Tembo provides connection-string through instance **Home** tab.
+
+```
+export PGPASSWORD=TEMBO_GENERATED_PGPASSWORD
+export PGHOST=TEMBO_GENERATED_PGHOST
+```
+
+## Data loading
+Data loaded remotely using psql client.
+
+**psql** installation
+
+```
+sudo apt-get update
+sudo apt-get install -y postgresql-client
+
+```
+**Data** download
+
+```
+wget --no-verbose --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz'
+
+gzip -d hits.tsv.gz
+```
+
+**Data** loading
+
+```
+psql postgresql://postgres:$PGPASSWORD@$PGHOST:5432 -t -c 'CREATE DATABASE test'
+
+psql postgresql://postgres:$PGPASSWORD@$PGHOST:5432/test -t < create.sql
```

Comment:
This fails with 
```
❯ psql postgresql://postgres:$PGPASSWORD@$PGHOST:5432/test -t < create.sql
ERROR:  access method "columnar" does not exist
```

Reply:
Same result. My instance:

```
https://cloud.tembo.io/orgs/org_2cpZgXjK2Ke7YOKb8T91DVEeTi5/clusters/inst_1708811361385_GzxAkc_8/home
```

Reply:
Hi @chhetripradeep and @tinybit ,
The instance mentioned by @tinybit is also OLAP stack which is the correct one. I was under the assumption that instance stack was wrong..whereas it wasn't the case. Real bad guy was my create script,  I removed couple of DDL from the top leads to this regression. Now it's fixed.. It show work fine. 
Thanks Guys for the good catch.

Reply:
Thanks for the quick update! We will continue with the review and hopefully finish it today